### PR TITLE
changing the experiment version for confluent = false

### PR DESCRIPTION
**Pull request by DeepikaNMS**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
 
image

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by DeepikaNMS** [comment]
@microsoft-github-policy-service agree company="Microsoft"

________________________________

File: src/confluent/azext_confluent/manual/commands.py [hunk 1]
```diff
@@ -35,3 +35,6 @@ def load_command_table(self, _):
     with self.command_group('confluent terms', confluent_marketplace_agreement,
                             client_factory=cf_marketplace_agreement, deprecate_info=g.deprecate(redirect='az term', hide=True)) as g:
         g.custom_command('list', 'confluent_terms_list')
+    
+    with self.command_group('confluent', is_experimental=False):
```
Comment (jsntcy) [inline src/confluent/azext_confluent/manual/commands.py:39]: No need to add this as by default `is_experimental` is false.

**Comment by jsntcy** [comment]
@DeepikaNMS, please fix the CI error first.

File: src/confluent/azext_confluent/manual/commands.py (hunk 1 above)
Comment (DeepikaNMS) [reply src/confluent/azext_confluent/manual/commands.py]: Ok, I have made the changes and have pushed the same !

File: src/confluent/azext_confluent/manual/commands.py [hunk 2]
```diff
@@ -35,3 +35,5 @@ def load_command_table(self, _):
     with self.command_group('confluent terms', confluent_marketplace_agreement,
                             client_factory=cf_marketplace_agreement, deprecate_info=g.deprecate(redirect='az term', hide=True)) as g:
         g.custom_command('list', 'confluent_terms_list')
+    with self.command_group('confluent'):
```
Comment (jsntcy) [inline src/confluent/azext_confluent/manual/commands.py:38]: I mean you should remove this line.

Comment (DeepikaNMS) [reply src/confluent/azext_confluent/manual/commands.py]: I have done this and pushed it. After that in the build checks

Comment (DeepikaNMS) [reply src/confluent/azext_confluent/manual/commands.py]: @jsntcy : Can you help by retriggering the checks and see if the same error occurs in cicd pipeline ?

**Comment by DeepikaNMS** [comment]
@jsntcy  : Can you review the PR? The checks have passed now.

**Comment by DeepikaNMS** [comment]
@zhoxing-ms, @kairu-ms  : Can one of you please review the PR

**Review by yonzhan — APPROVED** [review]

**Review by jsntcy — APPROVED** [review]

**Comment by jsntcy** [comment]
@DeepikaNMS, you need remove https://github.com/Azure/azure-cli-extensions/blob/main/src/confluent/azext_confluent/generated/commands.py#L41 in "generated" folder as well.

image

**Comment by DeepikaNMS** [comment]
> /azp run

**Comment by jsntcy** [comment]
@DeepikaNMS, please run `azdev test test_confluent_term_accept_flow --live` locally to regenerate the recording file (yaml file), then update the PR (The root cause is that cli core has just bump the api version for `resource` provider while your extension depends on it.)

**Comment by DeepikaNMS** [comment]
@jsntcy  : Got this error when I run the cmd azdev test test_confluent_term_accept_flow --live  and azdev test test_confluent_term_accept_flow --live --discover 

RUNNING TESTS LIVE
'test_confluent_term_accept_flow' not found. If newly added, re-run with --discover
No tests selected to run.

**Comment by jsntcy** [comment]
@DeepikaNMS, try to use `azdev setup` to reset the virtual environment and then run `azdev test test_confluent_term_accept_flow --live --discover` again to see if the issue still exists.

**Comment by DeepikaNMS** [comment]
Hi, 
I am now getting another error. When I googled, it asks me to create a subscription and I'm unable to 

`E           azure.core.exceptions.HttpResponseError: (AuthorizationFailed) The client 'deepikan@microsoft.com' with object id 'a16e04c3-322e-4895-9274-b9c8f94dabb1' does not have authorization to perform action 'Microsoft.Resources/subscriptions/resourcegroups/write' over scope '/subscriptions/a1ffc958-d2c7-493e-9f1e-125a0477f536/resourcegroups/cli_test_term_accept_basic_flowzhrzt5etraufqpsmr6tsel2ypowyhyo7hdorhsciwttc' or the scope is invalid. If access was recently granted, please refresh your credentials.
E           Code: AuthorizationFailed
E           Message: The client 'deepikan@microsoft.com' with object id 'a16e04c3-322e-4895-9274-b9c8f94dabb1' does not have authorization to perform action 'Microsoft.Resources/subscriptions/resourcegroups/write' over scope '/subscriptions/a1ffc958-d2c7-493e-9f1e-125a0477f536/resourcegroups/cli_test_term_accept_basic_flowzhrzt5etraufqpsmr6tsel2ypowyhyo7hdorhsciwttc' or the scope is invalid. If access was recently granted, please refresh your credentials.

azurecli-env\lib\site-packages\azure\mgmt\resource\resources\v2022_09_01\operations\_operations.py:11425: HttpResponseError`


Can you help ?

**Comment by jsntcy** [comment]
@DeepikaNMS, it seems you don't have permission for the operation for the subscription, please check with your team to get the permission for the operation.

**Comment by DeepikaNMS** [comment]
@jsntcy : Thanks, The tests worked and I have attached the recording.
[test_term_accept_basic_flow.zip](https://github.com/Azure/azure-cli-extensions/files/11061464/test_term_accept_basic_flow.zip)

**Review by jsntcy — APPROVED** [review]

**Comment by jsntcy** [comment]
@DeepikaNMS, just forgot one thing, you need increase extension version in setup.py and HISTORY.rst as well so that we can release a new version with your changes.

**Comment by DeepikaNMS** [comment]
@jsntcy : I have updated the files

**Comment by DeepikaNMS** [comment]
@jsntcy  : When can it be tested in Production ?

**Pull request merged**